process/outportBlockConverter_test.go

```diff
@@ -46,6 +56,102 @@ const (
 	outportBlockMetaBlockJSONPath = "../testscommon/testdata/outportBlockMetaBlock.json"
 )
 
+func TestOutportBlockConverter(t *testing.T) {
+	jsonBytes, err := os.ReadFile("../testscommon/testdata/outportBlocks.json")
+	require.NoError(t, err, "failed to read test data")
+
+	ob := &outport.OutportBlock{}
+	err = json.Unmarshal(jsonBytes, ob)
+	require.NoError(t, err, "failed to unmarshal test block")
+
+	converter, err := process.NewOutportBlockConverter(gogoProtoMarshaller, protoMarshaller)
+	require.Nil(t, err)
+
+	shardOutportBlock, err := converter.HandleShardOutportBlockV2(ob)
+	if err != nil {
+		panic(err)
+	}
```

Comment:
please use instead `require.Nil(t, err)`

Reply:
same to all other places where panic is used

Reply:
yes, sorry for that. leftovers when I was testing ad-hoc.